lib/pure/asyncdispatch.nim

```diff
@@ -42,12 +42,12 @@
 ##
 ## Code to read some data from a socket may look something like this:
 ##
-##   .. code-block:: Nim
-##      var future = socket.recv(100)
-##      future.addCallback(
-##        proc () =
-##          echo(future.read)
-##      )
+## .. code-block:: Nim
+##    var future = socket.recv(100)
+##    future.addCallback(
+##      proc () =
+##        echo(future.read)
+##    )
```

Comment:
Why not just change it to `runnableExamples`? Ditto for the other code-blocks.

Reply:
As one wise man below said: "previous way was fine" :P

Reply:
What? There are clear advantages of `runnableExamples` (see the contributing guidelines), it's not just different formatting.